x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -1158,6 +1159,9 @@ export class EndpointDocGenerator {
           version: '0.5.0',
           internal: false,
           removable: false,
+          install_version: '0.5.0',
```

Comment:
@paul-tavares I had to update this, is this ok?

Reply:
@neptunian yeah, this should be fine 😄